packages/next/src/server/lib/streaming-metadata.ts

```diff
@@ -23,3 +27,14 @@ export function shouldServeStreamingMetadata(
     !!userAgent && !blockingMetadataUARegex.test(userAgent)
   )
 }
+
+// When the request UA is a html-limited bot, we should do a dynamic render.
+// In this case, postpone state is not sent.
+export function isHtmlBotRequestStreamingMetadata(
```

Comment:
should we give this function a new name since it is no longer coupled to streaming metadata?